server/routes/admin.js

```diff
@@ -0,0 +1,54 @@
+var express = require('express');
+var passport = require("passport");
+var router = express.Router();
+var userSchema = require('../models/userSchema');
+var alumniSchema = require('../models/alumniSchema');
+var adminSchema = require('../models/adminSchema');
+const studentSchema = require('../models/studentSchema');
+require('mongoose').Promise = global.Promise
+
+async function isAdmin(id) {
+    let admin = await adminSchema.findById(id)
+    let alumni = await alumniSchema.findById(id).populate('user')
+    return (admin !== null || (alumni && alumni.user.role.includes('ADMIN')))
+}
+
+router.get('/one/:id', passport.authenticate('jwt', {session: false}), async (req, res, next) => {
+    try {
+        const dbData = await adminSchema.findOne({_id: req.params.id})
+        res.json({'result' : dbData});
+    } catch (e) {
+        console.log("Error: util#oneAdmin", e);
+        res.status(500).send({'error' : e});
+    }
+});
+
+router.get('/allAlumni/:adminId', passport.authenticate('jwt', {session: false}), async (req, res) => {
+    let adminId = req.params.adminId
+    try {
+        if (!isAdmin(adminId)) {
+            throw new Error('Invalid Admin ID');
```

Comment:
let's send a `res.status(400).send({message: 'Invalid Admin ID'})` so the browser isn't stuck waiting on a response.